### PR TITLE
Add logic for the test data

### PR DESCRIPTION
**Pull request by petracihalova**
when custom data doesn't exist, the app will copy and display the test data
when github and gitlab token is not provided in the `.env` file, the open and merged PRs are not downloaded after clicking on the button

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1,17 +1,17 @@
 import os
 from pathlib import Path
 
-SERVICES_LINKS_EXAMPLE_PATH = Path("data/services_links_example.yml")
-SERVICES_LINKS_PATH = Path("data/services_links.yml")
+DATA_PATH_FOLDER = "./data"
```
Comment (frenzymadness) [inline config.py:4]: By converting this to Path, the file can be much simpler:

```python
DATA_PATH_FOLDER = Path("./data")

SERVICES_LINKS_PATH = DATA_PATH_FOLDER / "services_links.yml"
GITHUB_PR_LIST = DATA_PATH_FOLDER / "github_pr_list.json"
```
and so on.

Instances of Path can use `/` for joining paths.

File: routes/merged_pr_page.py [hunk 2]
```diff
@@ -8,8 +8,9 @@ def get_github_merged_pr(reload_data):
...
     """
-    if config.GITHUB_MERGED_PR_LIST.is_file() and not reload_data:
-        with open(config.GITHUB_MERGED_PR_LIST, mode="r", encoding="utf-8") as file:
-            return json.load(file)
+    if config.GITHUB_TOKEN:
+        if not config.GITHUB_MERGED_PR_LIST.is_file() or reload_data:
+            return github_api.get_merged_pull_request()
 
-    return github_api.get_merged_pull_request()
+    with open(config.GITHUB_MERGED_PR_LIST, mode="r", encoding="utf-8") as file:
```
Comment (frenzymadness) [inline routes/merged_pr_page.py:15]: This is gonna fail if I don't have the token and the file does not exist. Other pages handle this by returning an empty dict.

File: routes/overview_page.py [hunk 3]
```diff
@@ -7,6 +10,17 @@ def get_services_links():
     """
     Get services links from a file.
     """
-    if config.SERVICES_LINKS_PATH.is_file():
-        return yaml.safe_load(config.SERVICES_LINKS_PATH.read_text())
-    return yaml.safe_load(config.SERVICES_LINKS_EXAMPLE_PATH.read_text())
+    if not config.SERVICES_LINKS_PATH.is_file():
+        try:
+            source = f"{config.DATA_PATH_FOLDER}/test_data"
```
Comment (frenzymadness) [inline routes/overview_page.py:15]:
```suggestion
            source = config.DATA_PATH_FOLDER / "test_data"
```

**Review by frenzymadness — COMMENTED** [review]
I see a possible improvement for handling the paths. Other than that it looks good to me.

File: config.py (hunk 1 above)
Comment (petracihalova) [reply config.py]: the `/` hack is awesome !!! thank you

File: routes/merged_pr_page.py (hunk 2 above)
Comment (petracihalova) [reply routes/merged_pr_page.py:18]: good catch, thank you

**Comment by frenzymadness** [comment]
It seems that there are multiple PRs with the same commits.

**Review by frenzymadness — APPROVED** [review]

**Pull request merged**